TRANSLATING.md

```diff
@@ -0,0 +1,24 @@
+# Translating Autotune to your language
+
+We are using [i18n-js](https://github.com/fnando/i18n-js) to translate the user interface.
+This library provides translation facilites using the same mechanism as the Rails I18n API.
+
+
+## Getting started:
+
+Copy the file _config/locale/autotune.yml_ to _config/locale/autotune-mylang.yml_ and edit as needed. Keep in mind that
+you *need* to change the 'en:' key to 'mylang:' in order to avoid overwriting other languages.
```

Comment:
Absolutely yes. You can:

- add them (under the correct language) into config/locale/autotune.yml
- add them into another file under autotune/config/locale
- add them into another file under my_application/config/locale